metadata/scripts/platemap_vis.r

```diff
@@ -39,7 +41,9 @@ platemap_df_partial <- readr::read_csv(
 # Add new column for "condition" to plot on platemap for DMSO versus compound
 platemap_df_partial <- platemap_df_partial %>%
     mutate(condition = ifelse(treatment == "DMSO", "DMSO", 
-                              ifelse(grepl("^S", treatment), "compound", NA)))
+                              ifelse(grepl("^UCD", treatment), "compound", 
+                                    ifelse(treatment == "TGFRi", "TGFRi", NA))))
+
```

Comment:
Is there anyway to check if there are NAs after mutating here?

Reply:
There is likely a way! For now, I think this will work.